falcon_sqla/__init__.py

```diff
@@ -14,6 +14,6 @@
 
 from .manager import Manager
 
-__all__ = [
+__all__ = (
```

Comment:
Do we really need to change this? I'm also a bit of control freak myself loving to make everything immutable (what can be made immutable), however, [Python doc examples do use lists](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package).

So IMHO better to stick to that.